src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.HumanDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.lib.InjectHuman;
+import core.basesyntax.model.Bet;
+
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    private static BetDao betDao;
+
+    @InjectHuman
```

Comment:
@TarasHryts Було б круто використовувати тільки одну анотацію Inject
```suggestion
    @Inject
```